library/src/main/java/com/datatorrent/lib/db/jdbc/AbstractJdbcPollInputOperator.java

```diff
@@ -80,201 +93,48 @@
  * @tags database, sql, jdbc, partitionable,exactlyOnce
  */
 @Evolving
-public abstract class AbstractJdbcPollInputOperator<T> extends AbstractStoreInputOperator<T, JdbcStore>
-    implements ActivationListener<Context>, IdleTimeHandler, Partitioner<AbstractJdbcPollInputOperator<T>>
+public abstract class AbstractJdbcPollInputOperator<T> extends AbstractStoreInputOperator<T, JdbcStore> implements
+    ActivationListener<OperatorContext>, IdleTimeHandler, Partitioner<AbstractJdbcPollInputOperator<T>>
 {
+  private static int DEFAULT_QUEUE_CAPACITY = 4 * 1024 * 1024;
+  private static int DEFAULT_POLL_INTERVAL = 10 * 1000;
+  private static int DEFAULT_FETCH_SIZE = 20000;
   /**
    * poll interval in milliseconds
    */
-  private static int pollInterval = 10000;
+  private int pollInterval = DEFAULT_POLL_INTERVAL;
+  private int queueCapacity = DEFAULT_QUEUE_CAPACITY;
+  private int fetchSize = DEFAULT_FETCH_SIZE;
 
   @Min(1)
   private int partitionCount = 1;
   protected transient int operatorId;
-  protected transient boolean isReplayed;
-  protected transient boolean isPollable;
+  protected boolean isPollerPartition;
   protected int batchSize;
```

Comment:
batchSize is optional, let me add some default value to it. What's the suggestion?